### PR TITLE
BXMSDOC-6032-7.33.x: Add section to migrate EAP from 7.2 to 7.3

### PR DESCRIPTION
**Pull request by csherrar**
[PAM Preview|http://file.ork.redhat.com/~csherrar/BXMSDOC-6032/7.33.x/PAM/#patches-migrating-eap-proc]
[DM Preview|http://file.ork.redhat.com/~csherrar/BXMSDOC-6032/7.33.x/DM/#patches-migrating-eap-proc]

### CODE REVIEW

File: doc-content/enterprise-only/installation/patches-migrating-eap-proc.adoc [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+.Procedure
+. Download {EAP} 7.3 to each {EAP} instance:
+.. Navigate to the https://access.redhat.com/jbossnetwork/restricted/listSoftware.html[Software Downloads] page in the Red Hat Customer Portal (login required) and select the product and version from the drop-down options:
+* *Product: {EAP}*
+* *Version: 7.3*
+.. Click *Download* next to *{EAP_LONG} 7.3*. (`jboss-eap-7.3.0.zip`).
+. Extract the `jboss-eap-7.3.0.zip` file to the same base directory of {EAP} 7.2.
++
+Note: The base directory of the {EAP} installation is referred to as `__EAP_HOME__`.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/installation/patches-migrating-eap-proc.adoc:19]: Not standard note format.

NOTE: The.... (renders wrong as is).

File: doc-content/enterprise-only/installation/patches-migrating-eap-proc.adoc [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+. Execute `jboss-server-migration.sh` to migrate configurations and deployments from {EAP} 7.2 to {EAP} 7.3:
```
Comment (emmurphy1) [inline doc-content/enterprise-only/installation/patches-migrating-eap-proc.adoc:20]: Actually, I would say run:
Run the BLAH script to....

I don't think you execute a script. Also, I like to say what at thing is.

File: doc-content/enterprise-only/installation/patches-migrating-eap-proc.adoc [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
++
+----
+$ ./bin/jboss-server-migration.sh -s ../jboss-eap-7.2/
+----
++
+Answer `yes` to all questions except the following one, which you should answer `no`:
```
Comment (emmurphy1) [inline doc-content/enterprise-only/installation/patches-migrating-eap-proc.adoc:26]: ..the following question. Answer 'no' to this question:

File: doc-content/enterprise-only/installation/patches-migrating-eap-proc.adoc [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+This tool is not able to assert if the non-persistent deployments found are compatible with the target server, skip scanner's deployments migration?
+----
+
+. Copy the `.niogit` directory from {EAP} 7.2. For example:
++
+----
+$ cp -r ../jboss-eap-7.2/.niogit
+----
+
+. Change the `MaxMetaspaceSize` memory setting value to `1024m` in `bin/standalone.conf`. For example:
```
Comment (emmurphy1) [inline doc-content/enterprise-only/installation/patches-migrating-eap-proc.adoc:38]: Minor, but:
in the `bin/standalone.conf`file.

File: doc-content/enterprise-only/installation/patches-migrating-eap-proc.adoc [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
++
+----
+-XX:MaxMetaspaceSize=1024m
+----
+
+. Optionally, to migrate unmanaged {KIE_SERVER} instances configuration to {EAP}, do the following tasks:
```
Comment (emmurphy1) [inline doc-content/enterprise-only/installation/patches-migrating-eap-proc.adoc:44]: We agreed to do this:
Optional: To migrate....
...perform the following tasks:
... complete the following tasks:

instance configurations?

File: doc-content/enterprise-only/installation/patches-migrating-eap-proc.adoc [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
++
+[IMPORTANT]
+====
+Proceed with this section only if there are unmanaged {KIE_SERVER} instances configured in the migrated {EAP} server instance that need to be migrated.
+====
+.. Locate the {KIE_SERVER} state XML files for the unmanaged instances. The XML files should be located in the directory from where the {EAP} server is launched. These are usually named after the {KIE_SERVER} instance identifier, for example `example-kieserver.xml`. You can use `grep` to search for `kie-server-state`, for example:
++
+----
+$ grep -FR "<kie-server-state>"
```
Comment (emmurphy1) [inline doc-content/enterprise-only/installation/patches-migrating-eap-proc.adoc:53]: Are the <> correct here?

File: doc-content/enterprise-only/installation/patches-migrating-eap-proc.adoc [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+----
+.. Copy the servers' state files to the directory where you will launch the new {EAP} 7.3 instance.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/installation/patches-migrating-eap-proc.adoc:55]: Do you mean servers` (plural possessive) or did you mean server's (singular possessive)?

File: doc-content/enterprise-only/installation/patches-migrating-eap-proc.adoc (hunk 5 above)
Comment (csherrar) [reply doc-content/enterprise-only/installation/patches-migrating-eap-proc.adoc:44]: Done.

I may be mistaken. I thought we agreed on "Optionally, "?

File: doc-content/enterprise-only/installation/patches-migrating-eap-proc.adoc (hunk 7 above)
Comment (csherrar) [reply doc-content/enterprise-only/installation/patches-migrating-eap-proc.adoc:55]: Excellent question, I'll ask Alberto.

File: doc-content/enterprise-only/installation/patches-migrating-eap-proc.adoc (hunk 4 above)
Comment (csherrar) [reply doc-content/enterprise-only/installation/patches-migrating-eap-proc.adoc:38]: Done

File: doc-content/enterprise-only/installation/patches-migrating-eap-proc.adoc (hunk 6 above)
Comment (csherrar) [reply doc-content/enterprise-only/installation/patches-migrating-eap-proc.adoc:53]: Good question, I'll find out.

File: doc-content/enterprise-only/installation/patches-migrating-eap-proc.adoc [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+Answer `yes` to all questions except the following one, which you should answer `no`:
++
+----
+This tool is not able to assert if the non-persistent deployments found are compatible with the target server, skip scanner's deployments migration?
+----
+
+. Copy the `.niogit` directory from {EAP} 7.2. For example:
++
+----
+$ cp -r ../jboss-eap-7.2/.niogit
```
Comment (jomarko) [inline doc-content/enterprise-only/installation/patches-migrating-eap-proc.adoc:35]: `cp` command requires also destination argument

Comment (csherrar) [reply doc-content/enterprise-only/installation/patches-migrating-eap-proc.adoc:35]: @Jomarko Can you confirm the destination path please? Thanks!

Comment (jomarko) [reply doc-content/enterprise-only/installation/patches-migrating-eap-proc.adoc:35]: Sure, we have two options I think
- `cp -r ../jboss-eap-7.2/.niogit .`
- `cp -r ../jboss-eap-7.2/.niogit .niogit`

File: doc-content/enterprise-only/installation/patches-migrating-eap-proc.adoc (hunk 1 above)
Comment (csherrar) [reply doc-content/enterprise-only/installation/patches-migrating-eap-proc.adoc:19]: Done

File: doc-content/enterprise-only/installation/patches-migrating-eap-proc.adoc (hunk 2 above)
Comment (csherrar) [reply doc-content/enterprise-only/installation/patches-migrating-eap-proc.adoc:20]: Done

File: doc-content/enterprise-only/installation/patches-migrating-eap-proc.adoc (hunk 3 above)
Comment (csherrar) [reply doc-content/enterprise-only/installation/patches-migrating-eap-proc.adoc:26]: Done

File: doc-content/enterprise-only/installation/patches-migrating-eap-proc.adoc (hunk 5 above)
Comment (csherrar) [reply doc-content/enterprise-only/installation/patches-migrating-eap-proc.adoc:44]: Changing to Optional: following discussion.

File: doc-content/enterprise-only/installation/patches-migrating-eap-proc.adoc (hunk 7 above)
Comment (csherrar) [reply doc-content/enterprise-only/installation/patches-migrating-eap-proc.adoc:55]: Turned out to be singular possessive.

File: doc-content/enterprise-only/installation/patches-migrating-eap-proc.adoc (hunk 8 above)
Comment (csherrar) [reply doc-content/enterprise-only/installation/patches-migrating-eap-proc.adoc:35]: Thanks, done

File: doc-content/enterprise-only/installation/patches-migrating-eap-proc.adoc (hunk 6 above)
Comment (csherrar) [reply doc-content/enterprise-only/installation/patches-migrating-eap-proc.adoc:53]: Yes, this is correct and was verified.

**Comment by csherrar** [comment]
> minor update needed

Thanks Jozef and Emily!

**Pull request merged**